src/cgWrappers.hpp

```diff
@@ -40,7 +40,7 @@ class CgTokensHandle final {
 
     CXTranslationUnit tu() const { return m_tu; }
     unsigned size() const { return m_ntokens; }
-    CXToken* tokens() { return m_data; }
+    CXToken* tokens() const { return m_data; }
```

Comment:
Leaving out `const` was intentional here: I don't want to provide a mutable handle to data in a const object.

But thinking about it, libclang provides no way to modify a `CXToken` so it is immutable anyway, thus :+1: on that.